### DISCUSSION TITLE
The Role of Dual Antennas in TDD SISO Configurations with USRP B210

### DISCUSSION THREAD

**Discussion by primedev88** [srsran/srsRAN_Project#745] [General Help] [answered]
My question is i,n SISO config and n78 band ,although we are using TDD configuration which means an antenna will work as transmitter for some time then receiver for another time then why TX/RX and RX both  antennas are used even in TDD config in B210?

**Comment by fllay** [comment, marked as answer]
srsRAN does not utilizes internal switch of B210 so TX/RX will be used as TX only and the receiving signal will be fed into the RX port. There is discussion to utilize B210 switch. However, I prefer to separate TX and RX port since it is a lot easier to add a PA or LNA
